BookingProject/BookingWebAPI/BookingWebApi.Application/DTOs/AppUserDTOs/LoginDto.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BookingWebApi.Application.DTOs.AppUserDTOs
+{
+    public class LoginDto
```

Comment:
Better use records for your DTOs, they match DTO pattern better because of immutability